tests/test_hierarchy_helpers.hpp

```diff
@@ -76,28 +104,52 @@ class LinearXMaterialProperty final : public dealii::Function<dim>
 };
 
 template <int dim>
-class LinearMaterialProperty : public dealii::Function<dim>
+class LinearMaterialProperty : public Coefficient<dim>
```

Comment:
Can't this class be `final` again?

Reply:
Yeah I work on too many branches at the same time :/